### PR TITLE
155a fix build

### PR DESCRIPTION
**Pull request by ericpromislow**
This gets `kim build` working in rancher-desktop with a simple fix.

### CODE REVIEW

**Review by mook-as — COMMENTED** [review]
With this, I get a different error:
```
Error: failed to solve: rpc error: code = Unknown desc = failed to solve with frontend dockerfile.v0: failed to read dockerfile: failed to mount pf93naxl3gjmybcq6kf5de6d2: snapshot pf93naxl3gjmybcq6kf5de6d2 does not exist: not found
```

**Comment by ericpromislow** [comment]
Could be because of the minikube/kim problems. Can you try this on top of PR #192?
________________________________

**Review by mook-as — APPROVED** [review]
Okay, things work with #192 pulled in, but I should make a PR to make the boot2ctl image be versioned — because `image.iso` already existed, it wasn't pulling the new one (and hence things were still differently broken).

I was going to say to wait until #192 is merged, but then I realized that this isn't any _more_ broken than `main`… so merge away!

**Pull request merged**
